index.html

```diff
@@ -0,0 +1,693 @@
+            </nav>
+        </header>
+        <main>
+            <section id="highlights">
+                <div class="highlight" id="highlight-1">
+                    <div class="highlight-red-gradient mobile-hidden tablet-hidden"></div>
+                    <div class="highlight-big-logo mobile-hidden tablet-hidden">
+                        <img alt="The Next Web" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646im_/https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    </div>
+                    <div id="coverDate" class="cover-date mobile-hidden tablet-hidden">Tuesday — January 26, 2021<div></div></div>
+                    <div class="highlight-gradient">
+                        <div class="highlight-data-container">
+                            <h3 class="highlight-title highlight-title-1">Opal cuts off your apps’ internet access to help you stop doomscrolling</h3>
+                            <p>By Ivan Mehta
+                                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                                1 hour ago
+                                <i class="fa fa-share-alt" aria-hidden="true"></i>
+                                6
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="highlight" id="highlight-2">
+                    <div class="highlight-gradient">
+                        <div class="highlight-data-container">
+                            <h3 class="highlight-title">Google unveils privacy-friendly AI cookie killer</h3>
+                            <p>By Thomas Macaulay
+                                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                                2 hour ago
+                                <span class="highlight-shares-small"><i class="fa fa-share-alt" aria-hidden="true"></i>
+                                    4</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="highlight" id="highlight-3">
+                    <div class ="highlight-gradient">
+                        <div class="highlight-data-container">
+                            <h3 class="highlight-title">This Mac app automatically detects your duplicate files</h3>
+                            <p>By Ivan Mehta
+                                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                                2 hour ago
+                                <span class="highlight-shares-small"><i class="fa fa-share-alt" aria-hidden="true"></i>
+                                    5</span>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section id="latest-news">
+                <h2 class="section-title">Latest news ></h2>
```

Comment:
- [x] Title of the section header should be a link.
```suggestion
                <h2 class="section-title"><a href="#">Latest news></a></h2>
```